### PR TITLE
Readme branch

### PR DESCRIPTION
**Pull request by PineappleApplePinecone**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+###Taxi Service Project
```
Comment (AntonVolskyi) [inline README.md]:
```suggestion
### Taxi Service Project
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+This is a web app that has several options in web-browser 
+to work with Database of drivers, cars and manufacturers
+####With this app you can:
```
Comment (AntonVolskyi) [inline README.md]:
```suggestion
#### With this app you can:
```

File: README.md (hunk 1 above)
Comment (AntonVolskyi) [reply README.md]: If you do not add a space, it will be displayed as `###Taxi Service Project`

**Review by AntonVolskyi — CHANGES_REQUESTED** [review]

**Review by AntonVolskyi — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+### Taxi Service Project
+This is a web app that has several options in web-browser 
+to work with Database of drivers, cars and manufacturers
+#### With this app you can:
+1. Without registration:
+* Watch registration page
+* Watch add/drivers page
```
Comment (Nick97-git) [inline README.md]: rewrite it without using `Watch`

File: README.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+2. Application layer (service)
+3. Presentation layer (controllers)
+### List of Technologies:
+You can find needed dependencies in pom.xml file
+* JDK 11
+* Apache Tomcat (v9.0.55)
+* Log4j (v2.14.1)
+* MySQL (v8.0.22)
+* JDBC
+* Javax Servlet (v4.0.1)
```
Comment (Nick97-git) [inline README.md]:
```suggestion
* Servlet API (v4.0.1)
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+* JSTL (v1.2)
+* JSP
+* Maven
+* Maven Checkstyle Plugin
```
Comment (Nick97-git) [inline README.md]: remove this line

File: README.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+* HTML, CSS
+### Recommendations for setup
+1. Configure Apache Tomcat for your IDE
+2. Install MySQL and MySQL Workbench or use terminal
+3. Register new user in MySQL Connections
+4. Cope and paste SQL schema from resources/init_db.sql in MySQL Workbench or terminal
```
Comment (Nick97-git) [inline README.md]:
```suggestion
4. Create schema in MySQL Workbench or terminal using resources/init_db.sql 
```

**Review by Nick97-git — CHANGES_REQUESTED** [review]
- remove redundant `/` from urls like /cars/, /drivers/, /manufacturers/
- DeleteCarController, DeleteDriverController, DeleteManufacturerController -> remove /all from /cars/all, /drivers/all, /manufacturers/all url patterns
- ConnectionUtil -> remove TIME_ZONE constant and in URL as value add stub

File: README.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
+### Taxi Service Project
+This is a web app that has several options in web-browser 
+to work with Database of drivers, cars and manufacturers
+#### With this app you can:
+1. Without registration:
+* Registration page view
+* Add/drivers page view
```
Comment (Nick97-git) [inline README.md]:
```suggestion
* Login
* Register new driver
```

**Review by Nick97-git — CHANGES_REQUESTED** [review]
I don't see changes according to my comments not related to README

**Comment by Nick97-git** [comment]
what about these comments?
Screenshot 2021-12-16 at 09 40 00

File: src/main/java/taxi/util/ConnectionUtil.java [hunk 8]
```diff
@@ -1,16 +1,15 @@
...
 import java.sql.Connection;
 import java.sql.DriverManager;
 import java.sql.SQLException;
 import java.util.Properties;
 
 public class ConnectionUtil {
-    private static final String URL = "jdbc:mysql://localhost:3306/taxi";
-    private static final String USERNAME = "username";
-    private static final String PASSWORD = "Cool_password";
+    private static final String URL = "YOUR URL AND TIME ZONE";
```
Comment (Nick97-git) [inline src/main/java/taxi/util/ConnectionUtil.java:9]:
```suggestion
    private static final String URL = "YOUR URL";
```

**Review by Nick97-git — APPROVED** [review]

**Pull request merged**